sdk/core/Azure.Core/src/RequestContext.cs

```diff
@@ -28,7 +28,35 @@ public class RequestContext : PipelineOptions
         /// <summary>
         /// Controls under what conditions the operation raises an exception if the underlying response indicates a failure.
         /// </summary>
-        public ErrorOptions ErrorOptions { get; set; } = ErrorOptions.Default;
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ErrorOptions ErrorOptions
```

Comment:
I am hoping @KrzysztofCwalina helps us find a better solution for handling enum inheritance, but this is functional today.